setuptools_protobuf/__init__.py

```diff
@@ -99,6 +99,8 @@ def pyprojecttoml_config(dist: Distribution) -> None:
     clean = dist.get_command_class("clean")
     clean.sub_commands.insert(0, ("clean_protobuf", has_protobuf))
 
+    dist.protoc = None  # type: ignore
```

Comment:
The error in #30 was about no `protoc_version` attribute, not `protoc`. Will this fix that?